app/views/nsm/steps/disbursement_cost/edit.html.erb

```diff
@@ -7,19 +7,20 @@
     <h1 class="govuk-heading-l"><%= t('.heading') %></h1>
     <%= step_form @form_object do |f| %>
       <% if @form_object.other_disbursement_type? %>
-        <%= f.govuk_number_field :total_cost_without_vat, min: 1, step: 0.01, width: 5, label: { tag: 'strong' }, data: { selector: 'valueField', multipler: @form_object.multiplier } %>
+        <%= f.govuk_text_field :total_cost_without_vat, width: 5, label: { tag: 'strong' }, data: { selector: 'valueField', multipler: @form_object.multiplier }, inputmode: 'numeric' %>
```

Comment:
Sorry, another nit: As these are decimal fields the inputmode should be `decimal`, not `numeric`, so that on mobile the soft keypad includes a decimal point as an option